rector.php

```diff
@@ -71,18 +72,25 @@
     // is there a file you need to skip?
     $rectorConfig->skip([
         __DIR__ . '/system/Debug/Toolbar/Views/toolbar.tpl.php',
+        __DIR__ . '/system/Test/ControllerResponse.php',
```

Comment:
@kenjis the issue seems on `ConstructClassMethodToSetUpTestCaseRector`, see issue:

- https://github.com/rectorphp/rector/issues/8188

this can be checked on specific rule:

```suggestion
        \Rector\PHPUnit\CodeQuality\Rector\Class_\ConstructClassMethodToSetUpTestCaseRector::class => [
            __DIR__ . '/system/Test/ControllerResponse.php',
        ],
```

Reply:
@kenjis here skip by rule ` \Rector\PHPUnit\CodeQuality\Rector\Class_\ConstructClassMethodToSetUpTestCaseRector::class` can be applied

Reply:
@samsonasik Thank you. Done with 0276d89